app/src/main/res/layout/view_preference_logout.xml

```diff
@@ -43,4 +43,17 @@
 
     </LinearLayout>
 
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/accountVanishButton"
+        style="@style/App.Button.Text"
+        android:layout_width="wrap_content"
```

Comment:
Would it be possible to have `match_parent` for the width to keep the layout consistency?

Reply:
yep!